README.md

```diff
@@ -57,112 +51,29 @@ It packs in lots of features that will allow your E-Commerce business to scale i
 **For Developers**:
 Take advantage of two of the hottest frameworks used in this project -- Laravel and Vue.js -- both of which have been used in Bagisto.
 
-### Documentation
-
-#### Bagisto Documentation [https://devdocs.bagisto.com](https://devdocs.bagisto.com)
-
-### Requirements
-
-* **OS**: Ubuntu 16.04 LTS or higher / Windows 7 or Higher (WampServer / XAMPP).
-* **SERVER**: Apache 2 or NGINX.
-* **RAM**: 3 GB or higher.
-* **PHP**: 7.3 or higher.
-* **Processor**: Clock Cycle 1 Ghz or higher.
-* **For MySQL users**: 5.7.23 or higher.
-* **For MariaDB users**: 10.2.7 or Higher.
-* **Node**: 8.11.3 LTS or higher.
-* **Composer**: 1.6.5 or higher.
-
-### Installation and Configuration
-
-**1. You can install Bagisto by using the GUI installer.**
-
-##### a. Download zip from the link below:
-
-[Download the latest release](https://github.com/bagisto/bagisto/releases/latest)
-
-##### b. Extract the contents of zip and execute the project in your browser:
-
-~~~
-http(s)://localhost/bagisto/public
-~~~
-
-or
-
-~~~
-http(s)://example.com/public
-~~~
-
-**2. Or you can install Bagisto from your console.**
-
-##### Execute these commands below, in order
-
-~~~
-1. composer create-project bagisto/bagisto-standard
-~~~
-
-~~~
-2. php artisan bagisto:install
-~~~
-
-**To execute Bagisto**:
+## Documentation
+Do you want to know how to install Bagisto or do you like to know how you can develop your own extensions?  
```

Comment:
Do you want to explore Bagisto or do you like to know how you can develop your own extension?